FLEDGE.md

```diff
@@ -1136,13 +1146,9 @@ Note that the key fields are used by the browser both to verify the signature, a
 
 The browser ensures, using TLS, the authenticity and integrity of information provided to the auction through calls made directly to an ad tech's servers. This guarantee is not provided for data passed in `runAdAuction()`. To account for this, additional bids use the same HTTP response header interception mechanism that's already in use for the [Bidding & Auction response blob](FLEDGE_browser_bidding_and_auction_API.md#step-3-get-response-blobs-to-browser) and `directFromSellerSignals`.
 
-To use HTTP response headers to convey the additional bids, the request to fetch them will first need to specify the `adAuctionHeaders` fetch flag.
-
-```
-fetch("https://...", {adAuctionHeaders: true});
-```
+To use HTTP response headers to convey the additional bids, the request to fetch them will first need to specify the `adAuctionHeaders` fetch flag or `adAuctionHeaders` attribute on an iframe element. The procedure for specifying `adAuctionHeaders` on an fetch or iframe navigation is described in section [2.5.2 Using Response Headers](#252-using-response-headers), except that the response would use `Ad-Auction-Additional-Bid` response headers instead of `Ad-Auction-Signals`, and the caller of `runAdAuction()` would specify `additionalBids` on the auction config instead of or in addition to `directFromSellerSignalsHeaderAdSlot`.
```

Comment:
```suggestion
To use HTTP response headers to convey the additional bids, the request to fetch them will first need to specify the `adAuctionHeaders` Fetch flag or `adAuctionHeaders` attribute on an iframe element. The procedure for specifying `adAuctionHeaders` on a Fetch or an iframe navigation is described in section [2.5.2 Using Response Headers](#252-using-response-headers), except that the response would use `Ad-Auction-Additional-Bid` response headers instead of `Ad-Auction-Signals`, and the caller of `runAdAuction()` would specify `additionalBids` on the auction config instead of or in addition to `directFromSellerSignalsHeaderAdSlot`.
```